src/js/utils/language.js

```diff
@@ -25,7 +25,9 @@ const codeToLang = {
     id: 'Indonesian',
     ko: 'Korean',
     th: 'Thai',
-    vi: 'Vietnamese'
+    vi: 'Vietnamese',
+    // TODO: Deprecate and replace with nn and nb
+    no: 'Norwegian'
```

Comment:
This kind of enforces the use of "no" instead of nn and nb. Not really needed IMO as these strings are not used in/part of our localization feature.

Reply:
removed!